packages/components/src/popover/index.js

```diff
@@ -153,6 +154,14 @@ class Popover extends Component {
 		}
 	}
 
+	/**
+	 * Stop all onMouseDown events propagating further - they should only go to the popover
+ 	 * @param {string} event Event object
+	 */
+	stopEventPropagationOutsidePopover( event ) {
+		event.stopPropagation();
```

Comment:
This seems logical, but also impactful, I'd appreciate some thoughts. @afercia @aduth